examples/orchestrated-apps/pages/orchestrated/_component.vue

```diff
@@ -0,0 +1,22 @@
+<script>
+import FooterComponent from '~/components/FooterComponent.vue'
+import HeaderComponent from '~/components/HeaderComponent.vue'
+
+const componentMap = {
+  footer: FooterComponent,
+  header: HeaderComponent
+}
+export default {
+  layout: 'orchestrated',
+  render (h) {
+    const component = process.server
+      ? this.$route.params.component
+      : this.$root.$options.orchestrated
+    return h(componentMap[component], {
+      attrs: {
+        'data-orchestrated-app': component
```

Comment:
This is what we hook into to detect this scenario client side and hydrate apps individually